core/base_service_test.go

```diff
@@ -1003,6 +1003,87 @@ func TestExtConfigFromVCAP(t *testing.T) {
 	clearTestVCAP()
 }
 
+func TestConfigureServiceFromCredFile(t *testing.T) {
+	service, err := NewBaseService(
+		&ServiceOptions{
+			Authenticator: &NoAuthAuthenticator{},
+			URL:           "bad url",
+		}, "service_1", "service_1")
+	assert.Nil(t, err)
+	assert.NotNil(t, service)
+	assert.Equal(t, "bad url", service.Options.URL)
+	assert.Nil(t, service.Client.Transport)
+
+	pwd, _ := os.Getwd()
+	credentialFilePath := path.Join(pwd, "/../resources/my-credentials.env")
+	os.Setenv("IBM_CREDENTIALS_FILE", credentialFilePath)
+
+	err = service.ConfigureService("service5")
+	assert.Nil(t, err)
+	assert.NotNil(t, service)
+	assert.Equal(t, "https://service5/api", service.Options.URL)
+	assert.NotNil(t, service.Client.Transport)
+
+	os.Unsetenv("IBM_CREDENTIALS_FILE")
+}
+
+func TestConfigureServiceFromVCAP(t *testing.T) {
+	service, err := NewBaseService(
+		&ServiceOptions{
+			Authenticator: &NoAuthAuthenticator{},
+			URL:           "bad url",
+		}, "service2", "service2")
+	assert.Nil(t, err)
+	assert.NotNil(t, service)
+  assert.Equal(t, "bad url", service.Options.URL)
```

Comment:
let's fix the indentation just to be all proper and such :)
indent should be a tab character

Reply:
it's weird, I did notice that in my PR but in my editor indentation is fine. I'll see what's going on